db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,25 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+  def up
+    LookupValue.transaction do
+      LookupKey.unscoped.find_each do |key|
+        LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
+      end
+    end
```

Comment:
Let's try skipping validations first: pure sql is always dangerous, especially as we support multiple db engines